assets/xml/objects/object_jj.xml

```diff
@@ -0,0 +1,76 @@
+<Root>
+    <File Name="object_jj" Segment="6">
+
+        <!-- Jabu Jabu animation (waggling fins etc.) -->
+        <Animation Name="gJabuJabuAnim" Offset="0x1F4C"/>
+            
+        <!-- Jabu Jabu eye textures -->
+        <Texture Name="gJabuJabuEyeOpenTex" OutName="jabu_jabu_eye_open" Format="rgb5a1" Width="16" Height="32" Offset="0x7698"/>
+        <Texture Name="gJabuJabuEyeHalfTex" OutName="jabu_jabu_eye_half" Format="rgb5a1" Width="16" Height="32" Offset="0x7A98"/>
+        <Texture Name="gJabuJabuEyeClosedTex" OutName="jabu_jabu_eye_closed" Format="rgb5a1" Width="16" Height="32" Offset="0x7E98"/>
+
+        <!-- Jabu Jabu skeleton -->
+        <Skeleton Name="gJabuJabuSkel" Type="Flex" LimbType="Standard" Offset="0xB9A8"/>
+
+        <!-- Jabu Jabu limb displaylists -->
+        <DList Name="gJabuJabuBodyDL" Offset="0x5690"/>
+        <DList Name="gJabuJabuLeftUpperFlipperDL" Offset="0x55B0"/>
+        <DList Name="gJabuJabuLeftMidFlipperDL" Offset="0x54D0"/>
+        <DList Name="gJabuJabuLeftLowerFlipperDL" Offset="0x5400"/>
+        <DList Name="gJabuJabuHeadDL" Offset="0x4920"/>
+        <DList Name="gJabuJabuLeftFrillDL" Offset="0x4880"/>
+        <DList Name="gJabuJabuRightFrillDL" Offset="0x47E0"/>
+        <DList Name="gJabuJabuRightUpperFlipperDL" Offset="0x4700"/>
+        <DList Name="gJabuJabuRightMidFlipperDL" Offset="0x4648"/>
+        <DList Name="gJabuJabuRightLowerFlipperDL" Offset="0x4578"/>
+        <DList Name="gJabuJabuFlukeDL" Offset="0x44A0"/>
+        
+        <!-- Jabu Jabu limbs -->
+
+
+
+        <Limb Name="gJabuJabuBodyLimb" LimbType="Standard" Offset="0xB87C"/><!-- Limb 4 -->
+
+        <Limb Name="gJabuJabuLeftUpperFlipperLimb" LimbType="Standard" Offset="0xB894"/><!-- Limb 6 -->
+
+        <Limb Name="gJabuJabuLeftMidFlipperLimb" LimbType="Standard" Offset="0xB8AC"/><!-- Limb 8 -->
+        <Limb Name="gJabuJabuLeftLowerFlipperLimb" LimbType="Standard" Offset="0xB8B8"/><!-- Limb 9 -->
```

Comment:
not a huge deal but we can prob get rid of alot of the whitespace in this section to make it more compact

Reply:
I left those in there so it's clear where limbs are missing. I can put some unnamed placeholders in instead, but I'm reluctant to make it look like this section is a complete limb list.

Reply:
even a comment will do, just the whitespace looks like it was done on accident